src/Navigation/Notifications/Repository/Notify.php

```diff
@@ -771,7 +771,7 @@ public function createFromNotification(Entity\Notification $Notification): bool
 			$title = '"' . trim(str_replace("\n", " ", $title)) . '"';
 		}
 
-		// Some mail software relies on subject field for threading.
+		// Some mail software rely on subject field for threading.
```

Comment:
I know the quality of the English in the comments isn't perfect, but in this case the verb is correctly conjugated as "software" is an always singular noun. The only issue with this sentence is the missing article before "subject".
```suggestion
		// Some mail software relies on the subject field for threading.
```

Reply:
Ah, nice to know. So the conjunction is not depending on the meaning (since "Some mail software" is plural in this context for me) but "hard determined" to the countability of the noun? Love to learn about natural language by the way on git. 😃